vite.config.ts

```diff
@@ -33,9 +33,15 @@ export default defineConfig({
       external: [
         'react',
         'react-dom',
-        'styled-components',
```

Comment:
It seemed to be okay to remove this line but will double check when I create a RC release.